### ISSUE TITLE
undefined: imgui.TextUnformatted

### ISSUE THREAD

**Issue by r00tc0d3** [inkyblackness/imgui-go#2]
I checked out the latest master. When build I get:

    imgui-go/_examples/opengl3_example  
    go build
    ./main.go:59:4: undefined: imgui.TextUnformatted
    ./main.go:68:4: undefined: imgui.TextUnformatted
    ./main.go:75:4: undefined: imgui.TextUnformatted

Any idea what causes this?
When I change it to imgui.Text(...) it works

**Comment by dertseha** [comment]
Hello and thank you for your message!

Also thank you for spotting this. I failed to check back with the example after I refactored the names of the Text() functions.
Your fix is the one I'd do, yes.

Reason for this was that there was no point in keeping "TextUnformatted" as none of the other Text* functions will be exported via Go. (Internally, they essentially go all to the same formatting variant of ImGui). So, there's only Text() now.
